### ISSUE TITLE
Add support for Solarized color scheme

### ISSUE THREAD

**Issue by pensierinmusica** [Benvie/JavaScriptNext.tmLanguage#75]
Hi,

It'd be awesome if this syntax could work with the [Solarized color scheme](http://ethanschoonover.com/solarized).

There's already a version of Solarized on [Sublime Package Control](https://packagecontrol.io/packages/Solarized%20Color%20Scheme).

Unfortunately at the moment the two packages (JavaScript Next, and Solarized) don't work fine together. Any chances this could be fixed or supported soon?

Thanks! Cheers

**Comment by simonzack** [comment]
I don't think this can be fixed by this package alone. Solarized may need some changes too. But are there any specific issues you're after?

**Comment by pensierinmusica** [comment]
Hi @simonzack, thanks for the quick reply. There are a couple of annoying things that stand out, let me show you through screenshots.

Normal Sublime **JS syntax**:

![js](https://cloud.githubusercontent.com/assets/3594037/6097057/550e402c-afae-11e4-8076-829410b029ec.png)

Sublime JavaScript Next syntax:

![jsnext](https://cloud.githubusercontent.com/assets/3594037/6097058/56c96efa-afae-11e4-873d-a9e499826827.png)


As you can see using **JavaScript Next syntax** there are 2 problems:

1) Variable names (e.g. after "var" or "let") get highlighted with the same color (light blue) as the "declarators" themselves (i.e. the "var", or "let" keywords).

2) Object keys use the same dark grey as comments, whereas I think it's better to differntiate between the two and keep the darker tone only for comments.

Let me know your thoughts! Thanks

**Comment by simonzack** [comment]
The variable issue is caused by the default package highlighting all `variable.other` scopes in the same color. `variable.other.readwrite` is used here compared to the default package where no scopes are added there at all, but we need this here to distinguish between different variable scopes. So I think this case it's better to fix monokai. Perhaps it will be easier to use some of the extended themes and patch those, like https://github.com/jonschlinkert/sublime-monokai-extended ?

**Comment by Snack-X** [comment]
I'm trying to land new features of JavaScriptNext to [Solarized on Package Control](https://github.com/braver/Solarized).
Currently, I only [fixed variable issues](https://github.com/Snack-X/Solarized/commit/f89527348d2b728dfe022845d856c28a0f168b12), and it's ok for now. Many issues are still not fixed, and will try fix it as soon as I discover.

**Comment by pensierinmusica** [comment]
@Snack-X it would be awesome if you could apply the same patches to ST3 default Solarized color scheme too (imho, it delivers better background contrast). Thanks!
